### ISSUE TITLE
Please clarify whether a new resource group gets created

### ISSUE THREAD

**Issue by v-kydela** [MicrosoftDocs/composer-docs#12]
It looks like the default is to use an existing RG, but the parameter table makes it sound like it creates a new RG:

"Name of your resource group | The name you give to the resource group you are creating."

Which is it?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b3462131-c17c-c9e1-106c-390e0bdb523d
* Version Independent ID: bd203f82-3725-6df6-1aef-9ffad92fa7fa
* Content: [Publish a bot to Azure - Bot Composer](https://docs.microsoft.com/en-us/composer/how-to-publish-bot)
* Content Source: [composer-docs/how-to-publish-bot.md](https://github.com/MicrosoftDocs/composer-docs-pr/blob/live/composer-docs/how-to-publish-bot.md)
* Service: **bot-composer**
* GitHub Login: @zxyanliu
* Microsoft Alias: **kamrani**

**Comment by zxyanliu** [comment]
The parameter table is for creating a new RG.

**Comment by zxyanliu** [comment]
I am wondering why did you think: "It looks like the default is to use an existing RG"?
If there is anything misleading like this, please point it out so that I can clarify. Thanks!

**Comment by v-kydela** [comment]
Lines 99 and 100 in provisionComposer.js:

```js
const templatePath =
  argv.customArmTemplate || path.join(__dirname, 'DeploymentTemplates', 'template-with-preexisting-rg.json');
```

Also the Azure Functions command specifies function-template-with-preexisting-rg.json explicitly

**Comment by zxyanliu** [comment]
I was having the same confusion regarding the Azure Functions scripts. Will get back to you after coordinating with the dev for an appropriate answer to your questions.

**Comment by v-kydela** [comment]
@zxyanliu - Follow the instructions in the [doc](https://docs.microsoft.com/en-us/composer/how-to-publish-bot#create-azure-resources):

> 1. Open a new Command Prompt. Navigate to the scripts folder of your bot's project folder. For example:
> 
> ```
> cd C:\Users\UserName\Documents\Composer\BotName\scripts
> ```

**Comment by v-kydela** [comment]
I've determined that the script creates a new resource group and then uses the preexisting-rg template. However, the name of the resource group actually gets the name of the "environment" appended to it. Not sure if you want to make that clear in the doc.

**Comment by zxyanliu** [comment]
Got it, you meant the `provisionComposer.js` file.

**Comment by v-kydela** [comment]
> Got it, you meant the `provisionComposer.js` file.

What did I say that was different from that?

**Comment by zxyanliu** [comment]
No, you are good. I was confused because of another issue on deployment.

**Comment by zxyanliu** [comment]
Checked with @luhan2017
* This is only a name, the template supports the creation of new resources in the existing resource group. 
In this script, there is a step that helps you create the resource group, so you do not need to worry about it at this point.
In the upcoming release of Composer, the publish experience is going to be different. So let's no worry about this now. 

Thanks again for your feedback!

**Comment by zxyanliu** [comment]
Hi, FYI, The publishing experience is going to be updated for R11 release (Nov), and docs will be updated accordingly.
Hopefully, the issues regarding publishing will all be addressed.
